Libraries/Image/RCTImageLoader.m

```diff
@@ -377,10 +371,24 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
-      }
 
-      // Call handler
-      completionHandler(nil, data);
+        NSString *responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
yea I briefly discussed that with @javache, our thinking was that since the server has to allow us to cache a request but is not providing a Date header, it is okay to cache it.

to be honest, this should *really* never show up, so I am also okay with ignoring checking the cache